src/vips/gvalue.lua

```diff
@@ -320,12 +320,12 @@ local gvalue_mt = {
                 local array = vips_lib.vips_value_get_array_image(gv, pint)
                 result = {}
                 for i = 0, pint[0] - 1 do
-                    -- this will make a new cdata object 
+                    -- this will make a new cdata object
                     local vimage = array[i]
 
                     -- vips_value_get_array_image() adds a ref for each image in
                     -- the array ... we must remember to drop them
-                    vobject.new(vimage)
+                    gobject_lib.g_object_ref(vimage)
```

Comment:
Can you explain this change? It looks like a ref leak to me.

Reply:
All other language bindings (except node-vips(?)) did this too, so I assumed it was the correct way. Without this change https://github.com/kleisauke/lua-vips/blob/cleanup/spec/gvalue_spec.lua#L71-L76 is failing.

node-vips: https://github.com/jcupitt/node-vips/blob/master/lib/gvalue.js#L281
ruby-vips: https://github.com/jcupitt/ruby-vips/blob/master/lib/vips/gvalue.rb#L209
pyvips: https://github.com/jcupitt/pyvips/blob/master/pyvips/gvalue.py#L301
net-vips: https://github.com/kleisauke/net-vips/blob/master/src/NetVips/GValue.cs#L479

Reply:
Oh huh then I'm sure you're right. Let's merge and I'll double-check.